docs/core/testing/unit-testing-mstest-writing-tests-attributes.md

```diff
@@ -29,7 +29,7 @@ Every test class must have the `TestClass` attribute, and every test method must
 
 ### `TestClassAttribute`
 
-The [TestClass](<xref:Microsoft.VisualStudio.TestTools.UnitTesting.TestClassAttribute>) attribute marks a class that contains tests and, optionally, initialize or cleanup methods.
+The <xref:Microsoft.VisualStudio.TestTools.UnitTesting.TestClassAttribute> attribute marks a class that contains tests and, optionally, initialize or cleanup methods.
```

Comment:
I actually prefer the original display here. It still uses an xref and the word "attribute" isn't repeated. (When you use an xref like that (with custom text), you don't actually need the angle brackets.)